drivers/gles2/rasterizer_canvas_gles2.cpp

```diff
@@ -1898,9 +1898,19 @@ void RasterizerCanvasGLES2::canvas_light_shadow_buffer_update(RID p_buffer, cons
 			}
 
 			state.canvas_shadow_shader.set_uniform(CanvasShadowShaderGLES2::WORLD_MATRIX, instance->xform_cache);
-			if (cull != instance->cull_cache) {
 
-				cull = instance->cull_cache;
+			VS::CanvasOccluderPolygonCullMode transformed_cull_cache = instance->cull_cache;
+
+			if (transformed_cull_cache != VS::CANVAS_OCCLUDER_POLYGON_CULL_DISABLED && p_light_xform.basis_determinant() * instance->xform_cache.basis_determinant() < 0) {
```

Comment:
Seems good, I would add parethesis () on the right of && for a bit more of clarity

Reply:
Done, threw in a newline as well for a good measure.